### PR TITLE
fixes for zsh installation

### PR DESCRIPTION
**Pull request by utku-caglayan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,14 @@
 * Go 1.15 or better
 
 ## Download & Install
-
+For bash:
```
Comment (yuce) [inline README.md:8]: How about: `### Bash`

File: README.md [hunk 2]
```diff
@@ -5,15 +5,14 @@
...
 ```
-curl https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/scripts/install_raw.sh | bash
+curl https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/scripts/install.sh | bash
 ```
 
-## Extended Download & Install
-
+For zsh:
```
Comment (yuce) [inline README.md:13]: How about: `### Zsh`

File: scripts/install.sh [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    echo "$addToPathDirectivesZSH"
+    echo
+fi
+curl --silent "https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/extras/zsh_completion.zsh" --output $HOME/.zsh_completion.sh
+mkdir -p $HZCLI_HOME/autocompletion/zsh
+mv $HOME/.zsh_completion.sh $HZCLI_HOME/autocompletion/zsh/hz-cli
+echo "$zshAutocompletionDirectives"
+
+echo
+echo "Installation for BASH:"
```
Comment (yuce) [inline scripts/install.sh:88]: Let's use: `Bash`

File: scripts/install.sh [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+if [ -z "$xdg_home" ]; then
+    # XDG_DATA_HOME was not set
+    xdg_home="$HOME/.local/share"
+fi
+bash_completion_dir="$BASH_COMPLETION_USER_DIR"
+if [ -z "$bash_completion_dir" ]; then
+    # BASH_COMPLETION_USER_DIR was not set
+    bash_completion_dir="$xdg_home/bash-completion"
+fi
+mkdir -p $bash_completion_dir/completions
```
Comment (yuce) [inline scripts/install.sh:103]: It's better to quote the arguments to guard against spaces in the variables. Also I would recommend writing the variable names in braces:
```
mkdir -p "${bash_completion_dir}/completions"
```

File: scripts/install.sh [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+mkdir -p $HZCLI_HOME/autocompletion/bash
```
Comment (yuce) [inline scripts/install.sh:104]: It's better to quote the arguments to guard against spaces in the variables. Also I would recommend writing the variable names in braces.

File: scripts/install.sh [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+curl --silent "https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/extras/bash_completion.sh" --output $HZCLI_HOME/autocompletion/bash/hz-cli
```
Comment (yuce) [inline scripts/install.sh:105]: The same comment about quotes and braces here for `--output`.

File: scripts/install.sh [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+ln -s $HZCLI_HOME/autocompletion/bash/hz-cli $bash_completion_dir/completions/hz-cli
```
Comment (yuce) [inline scripts/install.sh:106]: The same comment about quotes and braces here.

**Review by yuce — CHANGES_REQUESTED** [review]
Looks good, except few changes.

Also, it would be better if variables are quoted, since they may include spaces.

File: README.md (hunk 1 above)
Comment (utku-caglayan) [reply README.md]: Sure

File: README.md (hunk 2 above)
Comment (utku-caglayan) [reply README.md]: Sure

File: scripts/install.sh (hunk 3 above)
Comment (utku-caglayan) [reply scripts/install.sh]: Done, also to the Zsh log

**Review by yuce — APPROVED** [review]
Thanks, looks good!

**Pull request merged**